### DISCUSSION TITLE
GPUI Windows available?

### DISCUSSION THREAD

**Discussion by kingkong19** [zed-industries/zed#9531] [Feature Requests]
Now that zed is offering development builds for different OS, i gpui now avaliable to use on windows? (Readme still states it is MacOS only)

**Comment by ghost** [comment]
I believe it's still a WIP (https://github.com/zed-industries/zed/issues/5394) and some folks are hacking on getting a basic 'Hello World' application with GPUI running on Windows (see https://github.com/zed-industries/zed/issues/9540). There will likely be a release announcement once Windows support is officially available for GPUI!

**Comment by JosephTLyons** [comment]
While we don't have official Windows builds - Windows can currently be built from source. We will post an update when we are ready to back the claim that we are "Windows-ready."
